packages/models-library/src/models_library/rest_ordering.py

```diff
@@ -27,10 +27,10 @@ class OrderBy(BaseModel):
 
 
 class _BaseOrderQueryParams(RequestParameters):
-    order_by: OrderBy | None = None
+    order_by: OrderBy
```

Comment:
Q: why did this became mandatory non optional currently?

Reply:
@GitHK This class is used as a base for a model that is produce in a factory. The latter will override it and make it optional by injecting a default value.